components/script/dom/htmllinkelement.rs

```diff
@@ -377,6 +387,32 @@ impl HTMLLinkElementMethods for HTMLLinkElement {
     // https://html.spec.whatwg.org/multipage/#dom-link-target
     make_setter!(SetTarget, "target");
 
+    // https://html.spec.whatwg.org/multipage/#dom-link-crossorigin
```

Comment:
You're right. Could we share this code with htmlscriptelement.rs instead of duplicating it, in that case?